File: Javascript/easy/Exercism/rna-transcription/rna-transcription.js
export const toRna = (dnaStrand) => {
  if(dnaStrand.length === 1) { return dnaToRna[dnaStrand]; }
  let rna = '';
  for(let i = 0; i < dnaStrand.length; i++){
    rna += dnaToRna[dnaStrand[i]];
  }

  return rna;
};

export const dnaToRna = {
  G: 'C',
  C: 'G',
  T: 'A',
  A: 'U'
}

File: Python/easy/Exercism/rna-transcription/rna_transcription.py
def to_rna(dna_strand):
    dna_to_rna = {'G': 'C', 'C': 'G', 'T': 'A', 'A': 'U'}
    
    #rna_sequence = ""
    #for nucleotide in dna_sequence:
    #    rna_sequence += dna_to_rna[nucleotide]
    
    #return rna_sequence
    
    return ''.join(dna_to_rna[nucleotide] for nucleotide in dna_strand)
    
